lib/pages/chat/chat.dart

```diff
@@ -394,7 +361,9 @@ class ChatController extends State<Chat>
   void _tryLoadTimeline() async {
     loadTimelineFuture = _getTimeline();
     try {
-      await loadTimelineFuture;
+      await loadTimelineFuture?.then((_) {
+        _initializePinnedEvents();
+      });
```

Comment:
It has any impact on the displaying of message to chat list?

Reply:
NO